app/controllers/follows_controller.rb

```diff
@@ -25,6 +25,28 @@ def show
     end
   end
 
+  def bulk_show
+    skip_authorization
+
+    response = params[:ids].map(&:to_i).each_with_object({}) do |id, hsh|
+      hsh[id] = if current_user.id == id
```

Comment:
this breaks if I'm not logged in, I tried with:

```shell
curl -i "localhost:3000/follows/bulk_show?ids[]=1"
```

from the shell

Reply:
Interesting works incognito for me.  I'll look into it, we might just not call it when incognito.